torch/nn/intrinsic/modules/fused.py

```diff
@@ -1,52 +1,56 @@
 import torch
 from torch.nn import Conv1d, Conv2d, Conv3d, ReLU, Linear, BatchNorm1d, BatchNorm2d, BatchNorm3d
 
-class ConvReLU1d(torch.nn.Sequential):
-    r"""This is a sequential container which calls the Conv 1d and ReLU modules.
+# Used for identifying intrinsic modules used in quantization
+class _FusedModule(torch.nn.Sequential):
+    pass
```

Comment:
question: is having a common ancestor the right pattern? So far we were creating maps and list for that purpose

Reply:
yeah, i feel common parent is better than the list, what do you think?

Reply:
I don't have strong feelings either way: Having a common pattern feels better to me, but the lists/maps is the way we used to do

Reply:
yeah I have removed the list/maps in this PR, we can use isinstance(model, _FusedModule) to check for this in the future.